Gems/ROS2/Code/Source/RobotImporter/URDF/UrdfParser.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/std/string/string.h>
+#include <console_bridge/console.h>
+#include <sstream>
```

Comment:
The `sstream` include is not needed here.

Reply:
Done